### PR TITLE
Added an Open Source Article Link and solved Issue #701

### LINKED ISSUE #701: Update Forge Your Future with Open Source link
Our CI flags this as a redirect and is suggesting to use the full URL instead.

```diff
- - [Forge Your Future with Open Source](https://fossforge.com) ($) - Book devoted to explaining open source, how to find a project, and how to start contributing. Inclusive of all roles in software development, not just programmers.
+ - [Forge Your Future with Open Source](https://pragprog.com/titles/vbopens/forge-your-future-with-open-source/) ($) - Book devoted to explaining open source, how to find a project, and how to start contributing. Inclusive of all roles in software development, not just programmers.
```

### LINKED ISSUE #742: Article -> "Finding ways to contribute to open source on GitHub" Link added.
## Suggested link
https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github

## Reasoning
The Article is present in the Github Documentation. The page also has information related to topics such as "Finding Good First Issues" and "Opening a Pull Request"
## Section
This Link should be placed in the Readme File under "Useful articles for new Open Source contributors" section.

### PR DESCRIPTION
**Pull request by sanjeeban5644**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #742 and Closes #701

### CODE REVIEW

**Comment by sanjeeban5644** [comment]
### Hi! I am new to Open Source Contribution. I saw 4 issues and have tried to solve them.

### In the Last two commits, I have removed some Duplicate Links as there were errors while checks.

Screenshot (430)

### If I have done something incorrectly, please if you can correct me. Thanks! :)

File: README.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 
 ## Useful articles for new Open Source contributors
 
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github) by [@GitHub](https://github.com/github)
 - [How to choose (and contribute to) your first Open Source project](https://github.com/collections/choosing-projects) by [@GitHub](https://github.com/github)
 - [How to find your first Open Source bug to fix](https://www.freecodecamp.org/news/finding-your-first-open-source-project-or-bug-to-work-on-1712f651e5ba/) by [@Shubheksha](https://github.com/Shubheksha)
-- [First Timers Only](https://kentcdodds.com/blog/first-timers-only) by [@kentcdodds](https://github.com/kentcdodds)
```
Comment (erictleung) [inline README.md:119]: This can be added back and we can deal with the duplicate link later.

**Comment by erictleung** [comment]
@sanjeeban5644 thanks for taking the time to make this PR! All of it looks good. I think we can add back in that one link I commented on. Don't worry as much about the CI errors. The duplicate here is fine and we can deal with it in another issue. Thanks! Let me know if you have questions about this.

**Comment by sanjeeban5644** [comment]
I have made the changes you have asked for. Please let me know if I have done it correctly. :)

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking care of this issue and contributing to our repository! ✨

**Pull request merged**